File: src/main/java/com/EmosewaPixel/pixellib/recipes/EnergyRecipeList.java
package com.EmosewaPixel.pixellib.recipes;

import net.minecraft.util.ResourceLocation;

//This is a Recipe List for Energy Machine Recipes
public class EnergyRecipeList extends AbstractRecipeList<EnergyMachineRecipe, EnergyRecipeBuilder> {
    public EnergyRecipeList(ResourceLocation name, int maxInputs, int maxOutputs) {
        super(name, maxInputs, maxOutputs);
    }

    @Override
    public EnergyRecipeBuilder recipeBuilder() {
        return new EnergyRecipeBuilder(this);
    }
}

File: src/main/java/com/EmosewaPixel/pixellib/recipes/EnergyRecipeList.kt
package com.EmosewaPixel.pixellib.recipes

import net.minecraft.util.ResourceLocation

//This is a Recipe List for Energy Machine Recipes
class EnergyRecipeList(name: ResourceLocation, maxInputs: Int, maxOutputs: Int) : AbstractRecipeList<EnergyMachineRecipe, EnergyRecipeBuilder>(name, maxInputs, maxOutputs) {
    override fun recipeBuilder() = EnergyRecipeBuilder(this)
}